From 9a296cd6bf0e8a37630de0718fc098bbe57cdc61 Mon Sep 17 00:00:00 2001
From: Markus Niebel <Markus.Niebel@ew.tq-group.com>
Date: Thu, 4 Sep 2025 12:59:41 +0200
Subject: [PATCH] iMX95: remove unused targets from soc.mak

u-boot-atf.bin and u-boot-atf.itb are not valid for i.MX93 and depend
on the presence of mkimage_uboot and mkimage_fit_atf.sh. Remove.

Upstream-Status: Inappropriate

Signed-off-by: Markus Niebel <Markus.Niebel@ew.tq-group.com>
---
 iMX95/soc.mak | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/iMX95/soc.mak b/iMX95/soc.mak
index 326746d..e0e388c 100644
--- a/iMX95/soc.mak
+++ b/iMX95/soc.mak
@@ -278,17 +278,6 @@ u-boot-hash.bin: u-boot.bin
 	./$(MKIMG) -commit > head.hash
 	@cat u-boot.bin head.hash > u-boot-hash.bin
 
-u-boot-atf.bin: u-boot-hash.bin bl31.bin
-	@cp bl31.bin u-boot-atf.bin
-	@dd if=u-boot-hash.bin of=u-boot-atf.bin bs=1K seek=128
-
-u-boot-atf.itb: u-boot-hash.bin bl31.bin
-	./$(PAD_IMAGE) bl31.bin
-	./$(PAD_IMAGE) u-boot-hash.bin
-	TEE_LOAD_ADDR=$(TEE_LOAD_ADDR) ./mkimage_fit_atf.sh > u-boot.its;
-	./mkimage_uboot -E -p 0x3000 -f u-boot.its u-boot-atf.itb;
-	@rm -f u-boot.its
-
 u-boot-atf-container.img: bl31.bin u-boot-hash.bin
 	if [ -f $(TEE) ]; then \
 		if [ $(shell echo $(TEE_COMPRESS_ENABLE)) ]; then \
